docs/dev/news/5247.feat

```diff
@@ -0,0 +1 @@
+Add the attribute {:isolate_assertions}, which does the same as {:vcs_split_on_every_assert}.
```

Comment:
```suggestion
Add the attribute `{:isolate_assertions}`, which does the same as `{:vcs_split_on_every_assert}`. Deprecated `{:vcs_split_on_every_assert}`
```
I think we want to indicate to our users a preference towards the new attribute now. Could you perhaps add an "Info" or "Warning" if the second one is added, so that users know they should migrate? Keeping the two attributes for the same task might result in mixed codebases.

Reply:
Good idea. It would be easier to add this if we have a system for defining attributes and their types, and deprecation status. Would you mind if I don't make this change until we have such a system?

Reply:
I think it's fair to not do this change now but later. Also, we can delay this to the next release or afterwards.